### PR TITLE
issues/2050 add anduschain

### PR DESCRIPTION
**Pull request by metanonia**
Add Anduschain Mainnet

  "chainId": 14288640,
  "networkId": 14288640,

Thanks

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by metanonia** [comment]
I'm run prettier and re-deploy..
please check..

File: _data/chains/eip155-14288640.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "infoURL": "https://anduschain.io/",
+  "shortName": "anduschain-mainnet",
+  "chainId": 14288640,
+  "networkId": 14288640,
+  "explorers": [
+    {
+      "name": "anduschain explorer",
+      "url": "http://explorer.anduschain.io",
+      "icon": "daon",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-14288640.json:20]:
```suggestion
      "standard": "none"
```

the explorer does not seem to be EIP3091 compaible

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-14288640.json (hunk 1 above)
Comment (metanonia) [reply _data/chains/eip155-14288640.json]: I changed explorer standard and pushed.

It made new PR #2067

Comment (metanonia) [reply _data/chains/eip155-14288640.json]: #2067 was my mistake. I closed

Comment (metanonia) [reply _data/chains/eip155-14288640.json]: I changed explorer standard to none.
please check...

**Review by ligi — APPROVED** [review]

**Pull request merged**